_posts/2016-03-07-2016-forman-survey-analysis.md

```diff
@@ -0,0 +1,342 @@
+
+To those who said they spend active time on the lists and on IRC - *thank you!*
+User-to-user support is one of the things that makes community leads go all
+warm and fuzzy and liable to buy rounds ;)
+
+![User Support](/static/images/blog_images/2016-03-07-2016-forman-survey-analysis/user-support.png)
+
+We have a great reputation for our IRC channel, and that's reflected here in
+the 5-star rating our IRC support gets in the survey results. Happily the other
+support routes also do well, getting 4s across the board. This is great to see,
+but there's always more we can do, as seen below in the free-text comments.
+
+![Major and Minor releases](/static/images/blog_images/2016-03-07-2016-forman-survey-analysis/major-and-minor.png)
+
+I was woried about this one - if, say, frequency had come out as bad, we'd have
+no data on whether it was too fast or too slow, etc.
+
+Thankfully, we seem to do pretty well across the board, with the majority being
+3+ in every category, which is all the more important since it's largely the
+same as last year, showing that we have consistency in our releases as well as
+providing what our users want. Whilst a lot of people work on each release, I
+think it's worth calling out Dominic for all he does around release engineering
+- thanks!
+
+The final support question was a free text field, and I don't plan to
+cut'n'paste the entire result set, but we did get some themes. Out of the 17
+responses to this question:
+
+* 11 were generally positive about the support/docs (65%)
+* 7 requested more diagrams or how-to guides (potentially including screencasts) (41%)
+* 3 specifically requested improvements to plugin docs (18%)
+
+It's good to see two-thirds of the responses are positive, as a lot of effort
+goes into the manual and support channels. I still plan to produce my
+"beginners guide" screencast series "soon(tm)". Plugins, as ever, remain the
+responsibility of the authors, but the development community tries to help out
+where we can. Writing docs is also a great way to get [started with
+contributing][contribute], if you're so inclined!
+
+## <a name="foreman-features"></a>Specific Foreman Features
+
+Now we move onto the detail in some of Foreman's features:
+
+![Compute Resources](/static/images/blog_images/2016-03-07-2016-forman-survey-analysis/compute-resources.png)
+
+This graph is fascinating for quite a few reasons. Firstly, in a change from
+last year, we explicitly included "Physical hardware" as an option - although
+the question is about Compute Resources, in reality it should have been "What
+do you provision on with Foreman?". By including this separately from "None",
+we get that nearly two-thirds of the community stil manage physical hardware
+with Foreman. PXE is not dead yet, it seems!
+
+The other interesting figures are in the shifts from last year. EC2 and
+OpeStack both lose a few percentage points, while VMware and Docker (no
```

Comment:
Typo, OpenStack  - I think the reason they lost percentage points is likely due to Foreman not serving advanced user needs with regards to storage & networking in these 2 platforms, in case you want to mention it (I know it's kind of a negative statement..)